docs/docs/examples/logging/efk/EFK.md

```diff
@@ -5,44 +5,27 @@
 
-`Elasticsearch` is a distributed open search and analytics engine for all types of data. 
+[Elasticsearch](https://www.elastic.co/){.external} is a distributed open search and analytics engine for all types of data. 
 
-`Kibana` is an open-source front-end application that sits on top of Elasticsearch, providing search and data visualization capabilities for data indexed in Elasticsearch.
+[Kibana](https://www.elastic.co/kibana/){.external} is an open-source front-end application that sits on top of Elasticsearch, providing search and data visualization capabilities for data indexed in Elasticsearch.
 
-There are different methods to deploy an EFK stack. We provide two deployment methods, the first is deploying EFK locally on Kubernetes, and the second is using managed Elasticsearch outside the Kubernetes cluster. 
+There are different methods to deploy an EFK stack. We provide two deployment methods, the first is deploying EFK locally on Kubernetes, and the second is using a managed Elasticsearch instance outside the Kubernetes cluster. 
 
-## EFK using local Elasticsearch
+## Local EFK stack
 
-This solution deploys all of the EFK stack inside the Kubernetes cluster. By setting `fluentd.enabled` value to `true`, Helm installs Fluentd on each of application pods. This means that after deployment all the product pods run Fluentd, which collects all the log files and sends them to the Fluentd aggregator container. 
+This solution deploys the EFK stack inside the Kubernetes cluster. By setting `fluentd.enabled` value to `true`, Helm installs Fluentd on each of application pods. This means that after deployment all the product pods run Fluentd, which collects all the log files and sends them to the Fluentd aggregator container. 
 
 To complete the EFK stack you need to install an Elasticsearch cluster and Kibana, and successfully forward the aggregated datalog to Elasticsearch using Fluentd, which is already installed. 
 
-Follow these steps to install Elasticsearch:
+Follow these steps to install Elasticsearch
 
-```shell
-$ helm repo add elastic https://helm.elastic.co
-"elastic" has been added to your repositories
-
-$ helm install elasticsearch elastic/elasticsearch
-...
-```
+### 1. Install Elasticsearch
 
-Wait until all the nodes start and the status changes to `Running`:
-
-```shell
-$ kubectl get pods --namespace=dcd -l app=elasticsearch-master
-NAME                     READY   STATUS    RESTARTS   AGE
-elasticsearch-master-0   1/1     Running   0          100m
-elasticsearch-master-1   1/1     Running   0          100m
-elasticsearch-master-2   1/1     Running   0          100m
-$
-$ kubectl port-forward svc/elasticsearch-master 9200
-```
-Make sure Elasticsearch cluster is working as expected:
+Install Elasticsearch using the instructions [documented here](../../elasticsearch/BITBUCKET_ELASTICSEARCH.md). Once installed make sure Elasticsearch cluster is working as expected:
```

Comment:
The elastic installation pointed on this line is for Bitbucket search. This is very similar and it probably works, but in installing it adds an `image.tag` which is not needed when we provision an elasticsearch for EFK stack. 

Also we should change the `image.tag="7.9.3" in line 15 of `BITBUCKET_ELASTICSEARCH.md` as the default version for bitbucket is different in out helm chart (7.15.1)

Reply:
I actually think the `image.tag` is required, as taken from our ES install page:

> Choose a version of Elasticsearch that is supported by the [version of Bitbucket you are installing](https://confluence.atlassian.com/bitbucketserver/supported-platforms-776640981.html#Supportedplatforms-additional-toolsAdditionaltools)

Reply:
As for the `image.tag="7.9.3`, based on the current [app version defined in the BB Chart.yaml](https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/bitbucket/Chart.yaml#L6) i.e. `appVersion: 7.15.1-jdk11`, then Elasticsearch `7.9.3` [should be compatible](https://confluence.atlassian.com/bitbucketserver0715/supported-platforms-1082271274.html)?

Reply:
Right, We should have `image.tag` for elasticsearch in BB which currently the latest supported version is 7.10.2, but my point is this version is Bitbucket specific supported version. 
When we talk about EFK is a bit different because the elasticsearch is not interacting with our product directly so the limitation of BB for supporting versions does not applied. 

However, having 'image.tag' and the either `7.9.3` or `7.10.2` versions will works for EFK stack, but there is no logic behind that why we chosen this specific version for EFK and not any other version. The EFK stack works with ES 7.14 as well as version 7.9.

Reply:
I will leave it to you to decide if we need to redundant the installation commands or link to the existing one wrote for BB. The differences is pointing a specific versions which is needed for BB but is not for EFK stack.

Reply:
I think for consistency and simplification let's leave the tag as defined in the ES installation instructions this now points to.